src/MoBi.Core/Serialization/ORM/Mappers/ProjectMetaDataToProjectMapper.cs

```diff
@@ -61,11 +63,15 @@ public MoBiProject MapFrom(ProjectMetaData projectMetaData)
                }
 
                //then load the rest of the entities
+               var nonProjectBuildingBlocks = new List<IBuildingBlock>();
```

Comment:
Some new functionality here to implement a kind of project conversion.
If there's only one building block of each type, you get one module, otherwise we group by name and add them by name.